kusto/kusto.go

```diff
@@ -286,6 +280,17 @@ func setQueryOptions(ctx context.Context, op errors.Op, query Stmt, options ...Q
 			return nil, errors.ES(op, errors.KClientArgs, "QueryValues in the the Stmt were incorrect: %s", err).SetNoRetry()
 		}
 	}
+	if query.SupportsParameters() {
+		if len(opt.requestProperties.QueryParameters.ToParameterCollection()) != 0 {
```

Comment:
This needs to also be replaced with a simpler condition that does not require building the collection